primitives/arithmetic/src/per_things.rs

```diff
@@ -91,8 +96,10 @@ pub trait PerThing:
 	/// # }
 	/// ```
 	fn mul_floor<N>(self, b: N) -> N
-	where N: Clone + From<Self::Inner> + UniqueSaturatedInto<Self::Inner> + ops::Rem<N, Output=N> +
-		ops::Div<N, Output=N> + ops::Mul<N, Output=N> + ops::Add<N, Output=N> + Unsigned
+	where
```

Comment:
Sorry for changing the formatting, just did it manual bc the previous one was really not suited for multi where. Rest assured: we only remove the `From<Self::Inner>`